### DISCUSSION TITLE
Paid for by undefinded

### DISCUSSION THREAD

**Discussion by unional** [community/community#119829] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/3254987/3fea35e1-7c5b-43d6-90cc-2cb1f77df09a)

**Comment by Akash1134** [comment]
Hi @unional , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by unional** [reply]
Hi, I just noticed that in vscode. It seems to be fixed in 1.88.1
![image](https://github.com/community/community/assets/3254987/c885f80d-1fb1-4a1c-8ae8-faacb6354a97)
